### DISCUSSION TITLE
Changes in the Dao Section of Foundry Fundamentals

### DISCUSSION THREAD

**Discussion by WillStansill** [Cyfrin/foundry-full-course-cu#2133] [General]
working through the DAO section I noticed that when we use the OpenZeppelin Contract wizard to create our GovToken.sol file, things have changed quite a bit. 

![image](https://github.com/user-attachments/assets/d086b9d6-b789-4eb8-a089-ad236821a6a3)

This is about as close of a contract as we can compare from the video, and the first thing you will notice is that you will get an error that says Nonces is not found or not unique. 

To fix this issue simply just make sure that you import Nonces like so 

```
import {Nonces} from "../lib/openzeppelin-contracts/contracts/utils/Nonces.sol";
```
This solved my issue for me and I put a discussion on openzeppelin as well to let them know it needed to be added to the contract wizard

**Comment by EngrPips** [comment]
Yeah, Things have changed a little bit. Thanks for this heads-up.
